openapi.yml

```diff
@@ -394,9 +394,9 @@ paths:
 
 components:
   parameters:
-    EncodedPncId:
-      name: encodedPncId
-      description: A URL-encoded Police National Computer (PNC) identifier.
+    HmppsId:
+      name: hmppsId
+      description: A URL-encoded Police National Computer (HMPPS) identifier.
```

Comment:
This description still says Police National Computer - should probably just say "A URL-encoded HMPPS identifier."